package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We have this file in git ignore even though general practice checks this in. We are managing packages dynamically in our apps. Please delete.